media-gfx/curaengine/curaengine-2.3.1-r1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Commit message:
1. Please include full URL to upstream issue.
2. Please move `Bug` to footer rather than glued to body block.
3. Please use `Closes` instead to close the bug.

Also, please update the copyright line.